src/includes/getting-started-config/javascript.capacitor.mdx

```diff
@@ -0,0 +1,53 @@
+You should `init` the SDK as soon as possible during your app's creation stage and forward the `init` method from the sibling Sentry SDK for the framework you use.
```

Comment:
Some of this becomes redundant when you preview, since the common content includes basic prose. Edited to minimize the redundancy:
```suggestion
Then forward the `init` method from the sibling Sentry SDK for the framework you use, such as Angular in this example:
```